pom.xml

```diff
@@ -23,8 +23,11 @@
   </scm>
     <properties>
         <quarkus.version>3.15.3</quarkus.version>
+        <quarkus.angusmail.version>0.0.2</quarkus.angusmail.version>
         <quarkus.poi.version>2.1.1</quarkus.poi.version>
-        <tika.version>2.9.2</tika.version>
+        <quarkus.pdfbox.version>1.1.0</quarkus.pdfbox.version>
+        <commons-io.version>2.17.0</commons-io.version>
```

Comment:
This probably needs to be updated in the quarkus BOM too

Reply:
Ah quarkus-bom is at 2.18.0, nvm

Reply:
well we can use anything higher than 2.16.1 which is what was causing the issue?  Its all building NOW!

Reply:
2.18.0 should be OK?

Reply:
Maybe bump the Quarkus version to 3.18.3 and you get the upgrade for free

Reply:
3.18.2 did the trick!